ssvsigner/e2e/README.md

```diff
@@ -0,0 +1,90 @@
+# SSV-Signer E2E Tests
+
+End-to-end testing suite for SSV-Signer slashing protection functionality.
+
+## Overview
+
+This E2E suite tests all three slashing protection databases:
+
+1. **LocalKeyManager's BadgerDB** - Used when SSV Node signs directly
+2. **RemoteKeyManager's BadgerDB** - Used when SSV-Signer handles signing
+3. **Web3Signer's PostgreSQL** - Final protection layer for remote signing
+
+The tests validate that each database correctly prevents slashable operations in both signing paths:
+- **Local path**: SSV Node with LocalKeyManager → Direct signing
+- **Remote path**: SSV Node → SSV-Signer with RemoteKeyManager → Web3Signer
+
+## Test Environment
+
+- **Web3Signer 25.4.1** with PostgreSQL slashing protection database
+- **Docker containers** managed via testcontainers-go
+
+> **Note**: Database migrations in `testdata/migrations/` are copied from Web3Signer. When upgrading Web3Signer versions, check for new migrations at: https://github.com/ConsenSys/web3signer/tree/main/slashing-protection/src/main/resources/migrations/postgresql
+
+## Test Structure
+
+```
+e2e/
+├── suite.go                    # Shared E2ETestSuite
+├── common/                     # Utilities (keys, beacon objects, mocks)
+├── signing/                    # Test implementations
+│   ├── attestation_test.go     # Attestation slashing tests
+│   └── proposer_test.go        # Block proposal slashing tests
```

Comment:
should we add other duties? sync committee, etc..

Reply:
This PR: E2E tests for SSV-Signer slashing protection (attester + proposer).
Other signing domains (sync committee, etc.) are non-slashable but now easy to add with the established infrastructure.
Current scope covers all slashable operations.